variables.tf

```diff
@@ -60,6 +60,17 @@ variable "logdna_manager_key_name" {
   default     = "LogDnaManagerKey"
 }
 
+variable "logdna_resource_key_role" {
+  type        = string
+  description = "Role assigned to provide the IBM Cloud Logging key."
+  default     = "Manager"
+
+  validation {
+    condition     = contains(["Manager", "Reader", "Standard Member"], var.logdna_resource_key_role)
```

Comment:
I am not familiar with the Standard Member role. Could you please provide a link to the docs where you saw it, thanks?

Reply:
The service roles for IBM log analysis as per [this](https://cloud.ibm.com/iam/roles) are Manager, Reader and Standard Member. As well as this [doc](https://cloud.ibm.com/docs/log-analysis?topic=log-analysis-iam#service) also gives information about service roles.